doc/source/curry.rst

```diff
@@ -8,7 +8,7 @@ higher order function from ``functools``.  Currying provides syntactic sugar.
 .. code::
 
     >>> double = partial(mul, 2)    # Partial evaluation
-    >>> doubled = double(2)         # Currying
+    >>> doubled = double(5)         # Currying
```

Comment:
I feel the choice of passing in `2` might be slightly confusing for some people given the function itself is called `double`.